Sequences/qSlicerSequencesModule.h

```diff
@@ -33,8 +33,8 @@ qSlicerSequencesModule
   : public qSlicerLoadableModule
 {
   Q_OBJECT
-#ifdef Slicer_HAVE_QT5
-  Q_PLUGIN_METADATA(IID "org.slicer.modules.loadable.qSlicerLoadableModule/1.0");
+#ifdef Slicer_HAVE_QT5	
+  Q_PLUGIN_METADATA(IID "org.slicer.modules.loadable.qSlicerLoadableModule/1.0");	
```

Comment:
Hi @leochan2009,
Sorry! I either missed, or had forgotten about this pull request.

I'm not really familiar with Sequences, I can't comment on the logic.

If I did have a minor style nitpick, It would be to remove the trailing white-space from the lines in which it occurs.

Reply:
Thanks for your review!
I submitted this pull request to remind myself of the thing i did, otherwise i would probably delete my forked branch ;-)
Still a lot of codes need to be refined for the video streaming as Andras suggested.